### ISSUE TITLE
Allow users to submit their own challenges

### ISSUE THREAD

**Issue by Greenheart** [Limpan/parsons-puzzle#14]
This is an idea for the future.

## Why add this feature?
* This way, content can be provided by the users themselves.
* **We probably still want a similar tool** to allow admins to create challenges. Why not open it up for the community?

**Comment by Limpan** [comment]
User submitted content is great. It might need some kind of review process.

**Comment by Greenheart** [comment]
Yes - reviews will be necessary. Possibly done through the admin-UI?

**Comment by Limpan** [comment]
:+1:
